src/app_config.cpp

```diff
@@ -77,6 +82,7 @@ ConfigOpt *opts[] =
 // Wifi Network Strings
   new ConfigOptDefenition<String>(esid, "", "ssid", "ws"),
   new ConfigOptSecret(epass, "", "pass", "wp"),
+  new ConfigOptDefenition<String>(apass, "openevse", "apass", "apass"),
```

Comment:
The second name should be short to save flash space but the first can be a bit more descriptive
```suggestion
  new ConfigOptDefenition<String>(apass, "openevse", "ap_pass", "ps"),
```